DataRepo/models/hier_cached_model.py

```
@@ -90,7 +90,7 @@ def set_cache(rec, cache_func_name, value):
                 cache.set(rep_cachekey, rep_result, timeout=None, version=1)
     except Exception as e:
         # Allow tracebase to still work, just without caching
-        print(e)
+        print(f"{type(e).__name__}: {e}")
```

C. This is how I diagnosed the infusate cache issue that exists in main.  The way the cache exceptions are suppressed is not how I would do this given my current understanding, so this is due for a refactor, but I didn't want to add that feature creep to this effort.